ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+**Kind of issue**
+- [ ] Site request (add support for a new website)
+- [ ] Feature request (request new functionality)
+- [ ] Bug report (encountered problems with youtube-dl)
+- [ ] Other, namely ...
+
+**Run `youtube-dl --version` to check whether you have the latest version**
+- [ ] I've verified that I'm running the latest version available (2016.03.06)
+
+**Brief description of the problem/request**
+
+*I am having a problem with ... I have tried to do ... and ... I expected that ... would happen, but instead ... happened. Example: I tried to download a file but the site was not supported. Please add support for site xyz. Another example: I encountered a bug when downloading a video from xyz. I have tried to do a and b.*
+
+**Suggested solution or other information**
+
+*In case you have suggestions for a solution or any other relevant information you can write it here*
+
+**Example URL**
+
+*Provide the url of the file that you were trying to download.*
+
+**Output of running youtube-dl with `--verbose` or `-v`**
+```
+Replace the contents between the backticks (`) with the output of youtube-dl when running with the --verbose or -v flag.
+```
```

Comment:
This should go right after version check. There is no need to mention both `--verbose` and `-v` single `-v` is enough. It should be worded similarly to [this](https://github.com/rg3/youtube-dl#bugs) with example log between triple `'s. It should also mention early that applier is not required to fill this field it the issue's purpose is just a question or something else that does not imply providing a log.